.travis.yml

```diff
@@ -2,9 +2,9 @@ language: ruby
 cache: bundler
 rvm:
   - 2.1
-  - 2.2.4
-  - 2.3.1
-  - 2.4.1
+  - 2.2.8
+  - 2.3.5
+  - 2.4.2
```

Comment:
maybe just drop the z part so we're always testing with the latest version of each line?

Reply:
I'm ok with that but went this way to leave the project's existing style alone.  Some projects actively don't like to do it that way.  Occasionally it seems that Travis gets the logic behind that wrong - i.e., for a while this summer 2.2 was getting you 2.2.1 not 2.2.8 like you'd expect.

Reply:
@danielmorrison any preference?